rules/manager/manager.go

```diff
@@ -229,9 +236,11 @@ func (m *ruleManager) runIteration(results chan<- *extraction.Result) {
 					Timestamp: s.Timestamp,
 				}
 			}
-			m.results <- &extraction.Result{
-				Samples: samples,
-				Err:     err,
+
+			if err != nil {
+				evalFailures.Inc()
```

Comment:
Well, the only way this can error is if a rule times out or if the storage breaks somehow.

Syntactically incorrect rules are caught at startup time.